### DISCUSSION TITLE
Disable reverse DNS lookups for log entries

### DISCUSSION THREAD

**Discussion by simplydch** [yrutschle/sslh#479] [Q&A] [answered]
Hi there,

I'm having a bit of a problem with malicious actors and having trouble limiting them with fail2ban, I think this is because log entries are hostnames rather than IPs and the IP fail2ban returns from its DNS lookup.  Is there a way to disable the reverse DNS lookup and always print the IP address in the logs?

**Comment by yrutschle** [comment, marked as answer]
--numeric or equivalent in the configuration file?

30 nov. 2024 09:46:26 Darren Hunter ***@***.***>:

**Comment by simplydch** [comment]
Awesome! Thank you, and apologies if this was documented somewhere that I overlooked.
